src/component/update/index.tsx

```diff
@@ -24,7 +24,7 @@ interface UpdateProps extends DefaultUpdateProps {}
 type UpdateState = 'text' | 'graph' | undefined
 
 function Update(props: UpdateProps) {
-  const [updateView, setUpdateView] = useState<UpdateState>()
+  const [updateView, setUpdateView] = useState<UpdateState>('text')
```

Comment:
This is the fix to the bug we perceived where you had to click twice initially. 🙏